lib/rouge/lexers/graphql.rb

```diff
@@ -37,7 +37,7 @@ class GraphQL < RegexLexer
         rule %r/(""")(\n)(.*?)(\n)(""")/m do |m|
           token Str::Double, m[1]
           token Text::Whitespace, m[2]
-          delegate Markdown, m[3]
+          delegate Lexers::Markdown, m[3]
```

Comment:
Why do we need to qualify the namespace here? I'm not following the logic.

Reply:
Because `Markdown` may exist in the global scope, which means that the `Markdown` constant would resolve here and not trigger `const_missing`.

Reply:
Got it. Makes complete sense—thanks!